packs/equipment-effects/effect-weapon-siphon.json

```diff
@@ -0,0 +1,108 @@
+{
+    "_id": "mPb9Bhq8T1nsyWug",
+    "img": "systems/pf2e/icons/default-icons/effect.svg",
+    "name": "Effect: Weapon Siphon",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Weapon Siphon]</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "choices": {
+                    "ownedItems": true,
+                    "predicate": [
+                        "item:melee"
+                    ],
+                    "types": [
+                        "weapon"
+                    ]
+                },
+                "flag": "effectWeaponSiphonWeapon",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.Weapon"
+            },
+            {
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.WeaponSiphonDamage",
+                "option": "effect:weapon-siphon:damage",
+                "suboptions": [
```

Comment:
```suggestion
                "option": "effect:weapon-siphon:damage",
                "alwaysActive": true,
                "suboptions": [
```

Reply:
This is intentionally left false because there are three states the item can be in:

- Not attached to a weapon - nothing happens, don't apply the effect 
- Attached to a weapon, but no bomb loaded: you don't deal bonus damage, but the weapon siphon still worsens MAP, so you apply the effect but leave the toggle disabled. 
- Attached to a weapon, with a bomb loaded: you deal bonus damage and have worse MAP, so you apply the effect and toggle on the damage